### PR TITLE
chore(TDI-39921): Extend Integration tests for GoogleDrive

### PR DESCRIPTION
**Pull request by undx**
**What is the current behavior?** (You can also link to an open issue here)

https://jira.talendforge.org/browse/TDI-39921

**What is the new behavior?**

PR should be merged only when https://jira.talendforge.org/browse/DEVOPS-3763 is done.

**Please check if the PR fulfills these requirements**

- [X] The commit message follows Talend standard
- [X] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [X] The code coverage on new code >75%
- [X] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [X] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1674/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1674/)

File: components/components-googledrive/components-googledrive-integration/src/test/java/org/talend/components/google/drive/runtime/GoogleDriveBaseTestIT.java [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+        public Statement apply(final Statement base, Description desc) {
+            return new Statement() {
+
+                public void evaluate() throws Throwable {
+                    try {
+                        base.evaluate();
+                    } catch (SocketTimeoutException ex) {
+                        // If called with an expression evaluating to false, the test will halt and be ignored.
+                        Assume.assumeTrue(false);
```
Comment (khabali) [inline components/components-googledrive/components-googledrive-integration/src/test/java/org/talend/components/google/drive/runtime/GoogleDriveBaseTestIT.java]: This is useless in this scope as the exception is already catched and ignored.

I don't know why you need to ignore this error but please at least add a log of it.

Comment (undx) [reply components/components-googledrive/components-googledrive-integration/src/test/java/org/talend/components/google/drive/runtime/GoogleDriveBaseTestIT.java]: I agree that I should log it but this exception is not ignored and catch during tests.

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1681/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1681/)

File: components/components-googledrive/components-googledrive-integration/src/test/java/org/talend/components/google/drive/runtime/GoogleDriveBaseTestIT.java (hunk 1 above)
Comment (akhabali) [reply components/components-googledrive/components-googledrive-integration/src/test/java/org/talend/components/google/drive/runtime/GoogleDriveBaseTestIT.java]: Sorry I wasn't clear enough. I'm talking about the assume method call. It's useless as the exception is already catched and ignored.
So I think that you can remove this call

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1874/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1874/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/20/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/20/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/21/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/21/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/22/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/22/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Aundx%2FTDI-39921_GoogleDriveITs)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/30/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/30/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Aundx%2FTDI-39921_GoogleDriveITs)

**Pull request merged**